src/main/kotlin/com/github/bjornvester/wsdl2java/Wsdl2JavaTask.kt

```diff
@@ -21,8 +22,11 @@ open class Wsdl2JavaTask : DefaultTask() {
     @Optional
     var wsdlFiles: ConfigurableFileCollection = getWsdl2JavaExtension().wsdlFiles
 
+    @get:Input
+    @Optional
+    var bindingFile: Property<String> = getWsdl2JavaExtension().bindingFile
+
     @get:OutputDirectory
-    @get:PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Sorry for not getting back to you on this PR. What is the purpose behind removing the relative path sensitivity? It means that the output isn't relocatable, which disables the build cache if code is checked out on different paths (e.g. on a CI server).

Reply:
Ah, I see that OutputDirectory doesn't carry this semantics. Nevermind then :)